### PR TITLE
switch over to next-api

### PR DESCRIPTION
**Pull request by leggsimon**
This switches the content endpoint to use `next-api` instead of ElasticSearch. and removes unnecessary transforms

This also fixes the issue of the [Primary Theme link not linking](https://trello.com/c/EAGQC4kF/54-main-topic-not-linking-to-topic) as `primaryTag.relativeUrl` is now requested.

### CODE REVIEW

File: server/controllers/content.js [hunk 1]
```diff
@@ -20,13 +20,17 @@ router.get('/:uuid', (req, res, next) => {
...
-		.then(transformArticle({
-			showRelated: res.locals.flags.offlineArticleShowRelated
-		}))
+	const query = articleQuery(uuid)
+
+	fetch(`https://next-api.ft.com/v1/query?query=${query}&source=next-offline`, {
+			method: 'GET',
+			headers: { 'X-Api-Key': process.env.NEXT_API_KEY }
+		})
+		.then(response => response.json().data.article )
```
Comment (quarterto) [inline server/controllers/content.js]: This is throwing `cannot read property 'article' of undefined`. `response.json()` returns a promise so you need to do the `.data.article` in another `then`.

File: server/libs/transform-article.js [hunk 2]
```diff
@@ -5,10 +5,6 @@ const bylineTransform = require('../transforms/byline');
 
 const call = (...args) => fn => fn(...args);
 
-const selectPrimaryTag = article => {
-	article.primaryTag = article.metadata.find(tag => tag.primaryTag);
```
Comment (quarterto) [inline server/libs/transform-article.js]: I think you'll need to change the template to use `metadata.primaryTag` [here](https://github.com/Financial-Times/next-offline/blob/338f491436ef775a82decf1b7d5785b9010ab2f5/views/article.html#L12-L17)?

Comment (leggsimon) [inline server/libs/transform-article.js]: `primaryTag` stays on the top level as it is requested from next-api [here](https://github.com/Financial-Times/next-offline/pull/44/files#diff-82548b2f8f5a166a8cea308766a98eb2R27)

Comment (leggsimon) [inline server/libs/transform-article.js]: Although I can probably remove `primaryTag` from our metadata request

Comment (quarterto) [inline server/libs/transform-article.js]: oh haha i missed that

**Comment by leggsimon** [comment]
g2g?

**Pull request merged**